packages/text-html/src/HTMLText.ts

```diff
@@ -312,7 +313,7 @@ export class HTMLText extends Sprite
      * @private
      * @param {PIXI.CanvasRenderer} renderer - The renderer
      */
-    _renderCanvas(renderer: IRenderer)
+    _renderCanvas(renderer: CanvasRenderer)
```

Comment:
in `packages/canvas-display/global.d.ts` it is declared as the following:

```ts
        interface Container
        {
            _renderCanvas(renderer: import('@pixi/canvas-renderer').CanvasRenderer): void;
        }
```

which makes me think that this change is legit?